lib/compress/zstd_compress_internal.h

```diff
@@ -600,14 +577,30 @@ static void ZSTD_safecopyLiterals(BYTE* op, BYTE const* ip, BYTE const* const ie
     while (ip < iend) *op++ = *ip++;
 }
 
+#define ZSTD_REP_MOVE     (ZSTD_REP_NUM-1)
+#define STORE_REPCODE_1 STORE_REPCODE(1)
+#define STORE_REPCODE_2 STORE_REPCODE(2)
+#define STORE_REPCODE_3 STORE_REPCODE(3)
+#define STORE_REPCODE(r) (assert((r)>=1), assert((r)<=3), (r)-1)
+#define STORE_OFFSET(o)  (assert((o)>0), o + ZSTD_REP_MOVE)
+#define STORED_IS_OFFSET(o)  ((o) > ZSTD_REP_MOVE)
+#define STORED_IS_REPCODE(o) ((o) <= ZSTD_REP_MOVE)
+#define STORED_OFFSET(o)  (assert(STORED_IS_OFFSET(o)), (o)-ZSTD_REP_MOVE)
```

Comment:
`STORE_...` and `STORED_...` are easily confused. Better antonyms of `STORE_...` might be `LOAD_...` or `RETRIEVE_...` or `DECODE_...`.

Reply:
Agreed.
Initially, only the `STORE_*()` macros were supposed to exist.
The other ones (`STORED_*()`) were added later, when realizing that the numeric format of the sum type was relied upon in various parts of the code base. And then nb of `STORED_*()` macros kept increasing...

It's more a temporary situation.
In the following PR, macros names will be updated significantly.